vscode/microsoft-kiota/src/kiotaInterop/runtime.json

```diff
@@ -0,0 +1,29 @@
+{
+  "kiotaVersion": "1.22.3",
+  "runtimeDependencies": [
```

Comment:
we need to also update the CI scripts to it updates that file instead of the package.json

Reply:
True. I can add an update to do this.

Reply:
Done